omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -111,9 +111,10 @@
             delete "#{install_dir}/embedded/share/examples"
 
             # Setup pip aliases: `/opt/datadog-agent/embedded/bin/pip` will default to `pip2`
+            delete "#{install_dir}/embedded/bin/python"
```

Comment:
This would break backward compatibility on A6, in the situation where users use `#{install_dir}/embedded/bin/python` in some custom script (to install some extra wheels for a custom check for example).

Reply:
It's the first commit @KSerrania mentioned we should revert before merging, it's for testing purposes :)